japanese/toilet/toilet.md

```diff
@@ -0,0 +1,8 @@
+やっとトイレを見つけた、が、
+男子トイレと女子トイレのどちらに入るか迷ってしまう。
+
+あなたの性別は？
+
+[男](./male/male.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.